packages/x-data-grid/src/locales/frFR.ts

```diff
@@ -21,7 +21,7 @@ const frFRGrid: Partial<GridLocaleText> = {
   // Filters toolbar button text
   toolbarFilters: 'Filtres',
   toolbarFiltersLabel: 'Afficher les filtres',
-  toolbarFiltersTooltipHide: 'Cacher les filtres',
+  toolbarFiltersTooltipHide: 'Masquer les filtres',
```

Comment:
The change [makes sense](https://french.stackexchange.com/questions/8006/what-is-the-difference-between-cacher-and-masquer#:~:text=To%20me%2C%20it%20seems%20that,conceal%2C%20to%20store%20away%22.) to me.
But it would be interesting to double-check with @flaviendelangle or @alexfauquette 🤞

Reply:
I confirm it's more professional